src/vendor/github.com/xelabs/go-mysqlstack/sqlparser/ddl_test.go

```diff
@@ -128,6 +128,61 @@ func TestDDL1(t *testing.T) {
 				") engine=tokudb default charset=utf8",
 		},
 
+		// For issue: https://github.com/radondb/radon/issues/486
+		{
+			input: "create table test.t (\n" +
+				"	`id` int primary key,\n" +
+				"	`name` varchar(10)\n" +
+				") engine=tokudb comment='comment option' default charset=utf8 partition by hash(id)",
+			output: "create table test.t (\n" +
+				"	`id` int primary key,\n" +
+				"	`name` varchar(10)\n" +
+				") comment='comment option' engine=tokudb default charset=utf8",
+		},
+
+		{
+			input: "create table test.t (\n" +
+				"	`id` int primary key,\n" +
+				"	`name` varchar(10)\n" +
+				") engine='tokudb' comment='comment option' default charset='utf8' global",
+			output: "create table test.t (\n" +
+				"	`id` int primary key,\n" +
+				"	`name` varchar(10)\n" +
+				") comment='comment option' engine='tokudb' default charset='utf8'",
+		},
+
+		{
+			input: "create table test.t (\n" +
+				"	`id` int primary key,\n" +
+				"	`name` varchar(10)\n" +
+				") single engine=tokudb comment='comment option' default charset utf8 auto_increment 8",
```

Comment:
If the auto_increment 8 is not support, the output can not delete it.

Reply:
> If the `auto_increment 8` is not supported, the output can not delete it.

auto_increment option had been removed.